### ISSUE TITLE
Improved UI

### ISSUE THREAD

**Issue by bhavinjawade** [sh-dv/hat.sh#1]
I can design and implement a more dynamic, seamless and attractive UI for the application.
With a proper visualization of strength. 
Let me know if I should take up the issue.

**Comment by sh-dv** [comment]
yes , why not. but take in consideration that it should look as minimalistic as possible, we'll try to change it and update it if it looks better. preferred to use bootstrap.

**Comment by kriszrtina** [comment]
T

**Comment by kriszrtina** [comment]
https://github.com/sh-dv/hat.sh/issues/1#issuecomment-495787524
